charts/model-engine/templates/service_account_inference.yaml

```diff
@@ -1,25 +0,0 @@
-{{- if and (.Values.serviceTemplate) (.Values.serviceTemplate.createServiceAccount) (.Values.serviceTemplate.serviceAccountAnnotations) (.Values.serviceTemplate.serviceAccountName) (.Values.config.values.launch.endpoint_namespace)}}
```

Comment:
is there a reason we're removing this? seems like we could just add some more conditions to the if to achieve the same effect

Reply:
yea i'm fixing this- didn't realize this "createServiceAccount" was just for the inference account, which i was getting a naming conflict with so i removed for a quick fix

just updating with explicit naming, so we create another model-engine service account by default